support-frontend/app/services/pricing/DefaultPromotionService.scala

```diff
@@ -41,14 +43,20 @@ class DefaultPromotionServiceS3(
     extends DefaultPromotionService
     with LazyLogging {
 
+  // There is a DEV version of the default promos file, but CODE is much more use to us in regular use
```

Comment:
Could we just remove this?

My assumption was we were moving towards:
- `CODE` for everything besides `PROD`
- `PROD` for `PROD``
- If we need a version for testing, we use a local file?

Non-blocking

Reply:
Yes I would be inclined to get rid of it, not sure I can think of a use for it that you couldn't achieve by using code or a unit test.

Reply:
You might want to see things locally working for e2e testing - but having some sort of local file could work for that?